packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/LiveDataHostApiImpl.java

```diff
@@ -40,7 +40,7 @@ public LiveDataHostApiImpl(
   }
 
   /** Sets {@link LifecycleOwner} used to observe the camera state if so requested. */
-  public void setLifecycleOwner(@NonNull LifecycleOwner lifecycleOwner) {
+  public void setLifecycleOwner(@Nullable LifecycleOwner lifecycleOwner) {
     this.lifecycleOwner = lifecycleOwner;
```

Comment:
I would also annotate the field above with `@Nullable`, so that there are lint warnings when the value is used without a null check. Same goes for the other field in other classes.